.gitignore

```diff
@@ -1,9 +1,4 @@
-*.swp
-*.beam
-.DS_Store
```

Comment:
The above should be ignored, but in a "global" ignore file, usually located in the users home directory.